docs/documentation/index.html

```diff
@@ -2068,7 +2068,7 @@ <h1>Project Setup</h1>
 
 					<h2>Choose a Source Group Type</h2>
 
-					<p>When creating a new Source Group you can choose between different methods of setup depending on your project language. Please have a look at our <a href="#Choosingtherightprojectsetup">QUCICK START GUIDE section</a> for help on choosing the right Source Group type.</p>
+					<p>When creating a new Source Group you can choose between different methods of setup depending on your project language. Please have a look at our <a href="#ChoosingTheRightProjectSetup">QUICK START GUIDE section</a> for help on choosing the right Source Group type.</p>
```

Comment:
This anchor doesn't exist, nor did it exist when this content was added...

Reply:
Thanks for noticing. We will correct this in a different commit.